chatapplication/src/components/Searcharea.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './searcharea.css';
+
+
+const STYLES = "search-area"
+
+export const Searcharea = ({ 
+    type, 
+    onClick, 
+    searchStyle,
+    
+}) => {
+
+    const checkSearchStyle = STYLES.includes(searchStyle) ? searchStyle : STYLES[0];
+    return (
+        <div className='search-area-div'>
```

Comment:
I already know it is a div. I don't need to see it in the class name as well.
Maybe` search-area` or `search-area-wrapper` or `search-area-container `or smth else?